examples/benchmarking/M5-forecasting.ipynb

```diff
@@ -0,0 +1,3944 @@
+{
```

Comment:
<p>Before starting right with the features engineering description it would be nice do state the purpose of the notebook and also outline the next sections in this notebook.</p><p>The idea is that the reader knows at the beginning he will read something about feature engineering, afterwards about forecasting single items etc.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime/pull/7113/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>